documentation/docs/scenarios/xsa-deploy/Readme.md

```diff
@@ -0,0 +1,111 @@
+# Build and Deploy SAPUI5 or SAP Fiori Applications on HANA XS with Jenkins
+
+Build an application based on SAPUI5 or SAP Fiori with Jenkins and deploy the build result into an SAP Cloud Platform account in the Neo environment.
+
+## Prerequisites
+
+* TODO: do we have a general description how to setup docker. Doesn't make sense to describe such general setups on the level of each scenario.
+* You have installed the Java Runtime Environment 8. TODO: depends on docker setup.
+* You have installed Jenkins 2.60.3 or higher. TODO: not required in case CX Jenkins is used
+* You have set up Project “Piper”. See [README](https://github.com/SAP/jenkins-library/blob/master/README.md). TODO: not needed when CX Jenkins is used. Required for plain Jenkins use cases.
+* You have installed the Multi-Target Application (MTA) Archive Builder 1.0.6 or newer. See [SAP Development Tools](https://tools.hana.ondemand.com/#cloud). TODO: Obsolete with docker
+* You have installed Node.js including node and npm. See [Node.js](https://nodejs.org/en/download/). TODO: obsolete when using docker
+* TOOD: Prerequisite is that the artifacts we compile agaist are available either on Service Market Place (next TODO: explain how they can be imported into the build env), or via public maven repo
+
+### Project Prerequisites
+
+This scenario requires additional files in your project and in the execution environment on your Jenkins instance.
+
+On the project level, provide and adjust the following template:
+
+| File Name | Description | Position |
+|-----|-----|-----|
+| [`mta.yaml`](https://github.com/SAP/jenkins-library/blob/master/documentation/docs/scenarios/xsa-deploy/files/mta.yaml) | This file controls the behavior of the MTA toolset. | Place the `mta.yaml` file in your application root folder and adjust the values in brackets with your data. |
+
+## Context
+
+This scenario combines various different steps to create a complete pipeline.
+
+In this scenario, we want to show how to build a Multitarget Application (MTA) and deploy the build result into an on-prem SAP HANA XS advances system. This document comprises the [mtaBuild](https://sap.github.io/jenkins-library/steps/mtaBuild/) and the [xsDeploy](https://sap.github.io/jenkins-library/steps/xsDeploy/) steps.
+
+![This pipeline in Jenkins Blue Ocean](images/pipeline.jpg)
+###### Screenshot: Build and Deploy Process in Jenkins
+
+## Example
+
+### Jenkinsfile
+
+Following the convention for pipeline definitions, use a `Jenkinsfile` which resides in the root directory of your development sources.
+
+TODO: here we describe the straight-forward case. There is also a blue-green mode. How should we handle this
+
+```groovy
+@Library('piper-library-os') _
```

Comment:
Any negative impact when we change this to `piper-lib-os`?

Reply:
done